src/app/domain/iap/services/clouderyOffer.ts

```diff
@@ -0,0 +1,31 @@
+import { EventEmitter } from 'events'
+
+import { navigate } from '/libs/RootNavigation'
+
+import { Platform } from 'react-native'
+
+export const CLOUDERY_OFFER = 'CLOUDERY_OFFER'
+
+const START_IAP_REDIRECT_URL = 'https://iapflagship'
```

Comment:
Those strings look more like config than code, maybe we should put them somewhere else